bitcoin/tests/psbt.rs

```diff
@@ -206,8 +206,7 @@ fn create_transaction() -> Transaction {
 fn create_psbt(tx: Transaction) -> Psbt {
     // String from BIP 174 test vector.
     let expected_psbt_hex = include_str!("data/create_psbt_hex");
-    let expected_psbt = hex_psbt!(expected_psbt_hex).unwrap();
-
+    let expected_psbt: Psbt = hex_psbt(expected_psbt_hex);
```

Comment:
We don't need the type annotation here (and the others).

Reply:
It's good practice to use type annotations, although you're right, it's not needed.

Reply:
Reference please, I've never heard of this good practice and as far a I know we do not do that anywhere else unless its needed by the compiler.

Reply:
Note that Rust LSP is able to show the annotations without you having to write anything. I personally don't need it but I've seen a lot of people using it in neovim|VS Cod(e|ium).